lib/Segments/BulkAction.php

```diff
@@ -4,14 +4,16 @@
 
 use MailPoet\Listing\Handler;
 use MailPoet\Models\Segment;
-use MailPoet\WP\Hooks;
+use MailPoet\WP\Functions as WPFunctions;
```

Comment:
I'll ask here, but my comment applies to the whole PR. Why do you rename this function to `WPFunctions`. I find it a bit confusing. At first, I didn't notice and I was searching for `WPFunctions` in the DI config and it was not there. You use `$this->wp` in the whole file anyway.

Reply:
`use MailPoet\WP\Functions as WPFunctions` was used all across our codebase. So I tried to use the same to be consistent.